neural_network/simple_neural_network.py

```diff
@@ -0,0 +1,53 @@
+"""
+Forward propagation explanation:
+https://towardsdatascience.com/forward-propagation-in-neural-networks-simplified-math-and-code-version-bbcfef6f9250
+"""
+
+import random as ran
```

Comment:
Why are you importing it as `ran`?

Reply:
Personal preference, do you want me to change it?

Reply:
Its up to you, but I think consistency is key and from what I can tell the algorithms just `import random` normally